### DISCUSSION TITLE
How to update the pnpm?

### DISCUSSION THREAD

**Discussion by haosmos** [pnpm/pnpm#6003] [Q&A] [answered]
pnpm is updated very often. how to update the pnpm correctly and safely? Run the curl -fsSL installation command `curl -fsSL https://get.pnpm.io/install.sh | sh -` ? Or in some other way?

**Comment by weyert** [comment]
I think `pnpm add -g pnpm` should work?

**Comment by zkochan** [comment, marked as answer]
I recommend using the same script for updating pnpm that you used for installing it. So if you used the standalone script to install pnpm, then run the same script again and it will install the latest version of pnpm.

**Reply by haosmos** [reply]
> I recommend using the same script for updating pnpm that you used for installing it. So if you used the standalone script to install pnpm, then run the same script again and it will install the latest version of pnpm.

got it, thanks a lot! 👍

**Comment by paro-paro** [comment]
Hi @zkochan !

Thanks a lot for creating pnpm. Is absolutely great! My quick question... 

Would you consider shipping a new CLI command that automatically and conveniently takes care of this for us?

Something like `bun upgrade`

https://bun.sh/docs/installation#upgrading

I currently use an alias that runs a copy of the https://get.pnpm.io/install.sh script but without running the `setup` command at the end. 

I guess is a good workaround, but I'd love to have this option shipped officially.

Thanks!

P.S. I guess `upgrade` is not available since is an alias of `update` but whatever word fits best.
